src/includes/sentry-cli-sourcemaps.mdx

```diff
@@ -10,16 +10,12 @@ For more info on `sentry-cli` configuration visit the [Sentry CLI configuration
 
 Make sure `sentry-cli` is configured for your project. For that you can use environment variables:
 
-<Note>
-
-Visit the [auth token user settings page](https://sentry.io/settings/account/api/auth-tokens/) to find your auth tokens.
-
-</Note>
+<OrgAuthTokenNote />
 
 <SignInNote />
 
 ```bash {filename:.env.local}
-SENTRY_AUTH_TOKEN=your-auth-token
+SENTRY_AUTH_TOKEN=___ORG_AUTH_TOKEN___
 SENTRY_ORG=___ORG_SLUG___
 SENTRY_PROJECT=___PROJECT_SLUG___
```

Comment:
we should move always move this below the org/project slug, as in the UI they are kind of coupled together - selecting the org/project influences where you create the token for. So IMHO it is a better flow to always make this:
```suggestion
SENTRY_ORG=___ORG_SLUG___
SENTRY_PROJECT=___PROJECT_SLUG___
SENTRY_AUTH_TOKEN=___ORG_AUTH_TOKEN___
```